### ISSUE TITLE
Warning: dependency / bundle size issues

### ISSUE THREAD

**Issue by yougotashovel** [wrakky/react-html-parser#41]
TL;DR - use '[html-react-parser](https://github.com/remarkablemark/html-react-parser)' instead of 'react-html-parser' library to decrease bundle size.

I found using this package that it was increasing my bundle sizes dramatically.

Appears to be adding 182kb to bundle sizes, which is disproportionate for what this lib does. 

<img width="611" alt="size-increase" src="https://user-images.githubusercontent.com/523614/36174821-75595cc2-10db-11e8-801b-6811bae35040.png">

Here's an example of bundle sizes and included dependencies before and after a blank create-react-app included 'react-html-parser'

<img width="1276" alt="before" src="https://user-images.githubusercontent.com/523614/36174804-6cc0a296-10db-11e8-9942-8d2f884ac546.png">

<img width="1275" alt="after" src="https://user-images.githubusercontent.com/523614/36174813-70ed20ec-10db-11e8-9dbb-4ab2c273a9fe.png">
